pnpm-lock.yaml

```diff
@@ -1317,7 +1317,7 @@ importers:
         version: 0.13.4
       sass-loader:
         specifier: 12.4.0
-        version: 12.4.0(sass@1.54.0)(webpack@5.86.0)
```

Comment:
I think these sass changes in the lockfile can be avoided by upgrading your pnpm version